bin/pycbc_inspiral

```diff
@@ -238,10 +241,17 @@ with ctx:
         logging.info("Finished")
         sys.exit(0)
 
+    if opt.q_transform == True:
```

Comment:
You can just write if opt.enable_q_transform: also the next line should be an else. (What other options are there??).